### PR TITLE
refactor: replace deprecated toolbar methods (close #147)

### LINKED ISSUE #147: refactor addButton, addDivider in extensions
## Version

1.1.0-a

## Test Environment

all

## Current Behavior

ColorSync, ScrollSync 2 extensions depend on deprected methods(`addButton`, `addDivider`)

## Expected Behavior

refactor them to use `addItem`, `insertItem`.

### PR DESCRIPTION
**Pull request by kyuwoo-choi**
### Please check if the PR fulfils these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description
refactoring the deprected toolbar methods in extensions.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/extensions/scrollSync/scrollSync.js [hunk 1]
```diff
@@ -47,8 +47,10 @@ function scrollSyncExtension(editor) {
       $el: $(`<button class="active ${className}" type="button"></button>`)
     });
 
-    $divider = toolbar.addDivider();
-    toolbar.addButton(button);
+    toolbar.addItem('divider');
+    const items = toolbar.getItems();
+    $divider = items[items.length - 1].$el;
```
Comment (seonim-ryu) [inline src/js/extensions/scrollSync/scrollSync.js]: `items`가 0인 경우는 없나요?? 0일 때 `$el` 프로퍼티 없음 에러 발생하겠네요~

**Comment by jinwoo-kim-nhn** [comment]
리뷰 완료했습니다. 
colorButtonClicked, scrollSyncToggle 버튼의 위치만 변경된건가요?

File: src/js/extensions/scrollSync/scrollSync.js (hunk 1 above)
Comment (kyuwoo-choi) [reply src/js/extensions/scrollSync/scrollSync.js]: 적어도 바로 위에서 하나는 넣었으니 0일때는 없겠네요.

**Comment by kyuwoo-choi** [comment]
@jinwoo-kim-nhn deprecated된 addButton 바꾼거예요.

**Pull request merged**